source/tutorials_wine.rst

```diff
@@ -616,13 +616,15 @@ Paste code into the file:
    kind: ModelDeployment
    spec:
      image: "<fill-in>"
+     predictor: odahu-ml-server
```

Comment:
odahu-mlflow

Reply:
@dsuslov 
Our Flask wrapper basically works with GPPI format, no matter if MLFlow inside or not. We can name it `odahu-gppi-server`.